Zend/tests/attributes/028_grouped.phpt

```diff
@@ -0,0 +1,76 @@
+--TEST--
+Attributes can be grouped
+--FILE--
+<?php
+
+#[A1(1), A1(2), A2(3)]
```

Comment:
It would be good to also test that grouping works for attributes without an argument list.